packages/back-end/src/api/ingestion/ingestion.router.ts

```diff
@@ -43,9 +44,20 @@ export const getDataEnrichment = createApiRequestHandler({
     const dataSourcesByOrgId = Object.fromEntries(
       dataSources.map((ds) => [ds.organization, ds.id])
     );
+    const orgIds = Object.keys(dataSourcesByOrgId);
+
+    const orgIdsWithTrackingDisabled = await getOrganizationIdsWithTrackingDisabled(
+      orgIds
+    );
+
+    const orgIdsWiithTrackingEnabled = orgIds.filter(
+      (x) => !orgIdsWithTrackingDisabled.has(x)
```

Comment:
Instead of filtering them out, do we want to include them with a flag so we can return a more informative error in the ingestor instead of just "ClientKeyInvalid 400"?

Reply:
I did have that to begin with. I was going to return 403 forbidden. This seemed easier as I didn't have to touch the other repo. It is unclear how much we will use this and how much we will stick with putting them in the overages table so it makes sense to me to spend more time on that. Especially as we might want to just reach out to them by email if we ever actually use it